src/Config.hpp

```diff
@@ -16,6 +16,9 @@ struct Config {
     // 16-bit little-endian signed integers
     snd_pcm_format_t format { SND_PCM_FORMAT_S16_LE };
 
+    // Enable JSON formatted output
+    bool enable_json { false };
```

Comment:
If future we may want to add other output formats. So instead of a boolean, I suggest to add a enum, something like Formatting, FormatterType, OutputFormat, etc. with two values (for text and json).

Also, a nitpick: the fields access/format/sample_rate/channels are related and thus grouped together, so maybe better move the new filed to the beginning or to the end of the struct.